packages/utilities/psammead-test-helpers/package.json

```diff
@@ -25,7 +25,12 @@
     "snapshot"
   ],
   "dependencies": {
-    "jest-styled-components": "^6.3.1",
-    "react-test-renderer": "^16.6.3"
+    "jest-styled-components": "^6.3.1"
+  },
+  "devDependencies": {
+    "react": "^16.6.3",
```

Comment:
When I run this locally I dont need `react` and `react-dom`? Are you sure these are needed?